pmedian-precompile.jl

```diff
@@ -0,0 +1,2 @@
+import Benchmark
+Benchmark.run_paper_benchmark([1_0, 5_0, 10_0, 50_0])
```

Comment:
This is annoying if you have `cvxpy.json` already generated because it will fail

Why not just 
```Julia
import Benchmark
run_benchmark(
    num_facilities = 100,
    num_customers = 100,
    num_locations = 100,
    time_limit_sec = 0.001,
    max_iters = 1,
)
```

Reply:
New precompile is excellent though
```Julia
\multirow{3}{*}{1000} & generate & 0.06 & 0.15 & 0.05 & 1.51 & 0.48 & 0.05 & 0.06 & 1.38 \\
		& load & 0.20 & 0.43 & 0.06 & 0.26 & 0.72 & 0.17 & 0.11 & 0.10 \\
		& total & 0.27 & 0.58 & 0.12 & 1.77 & 1.20 & 0.22 & 0.17 & 1.48 \\
		\multirow{3}{*}{5000} & generate & 0.24 & 0.74 & 0.31 & 11.49 & 1.95 & 0.11 & 0.57 & 10.89 \\
		& load & 1.47 & 1.83 & 0.40 & 1.55 & 4.26 & 1.51 & 0.95 & 0.72 \\
		& total & 1.72 & 2.57 & 0.71 & 13.04 & 6.21 & 1.61 & 1.52 & 11.61 \\
		\multirow{3}{*}{10000} & generate & 0.74 & 1.45 & 0.69 & 30.27 & 4.09 & 0.28 & 0.94 & 29.31 \\
		& load & 2.76 & 4.09 & 0.77 & 2.77 & 8.90 & 3.44 & 1.96 & 1.42 \\
		& total & 3.50 & 5.53 & 1.45 & 33.04 & 12.98 & 3.72 & 2.90 & 30.73 \\
		\multirow{3}{*}{50000} & generate & 3.33 & 6.63 & 3.58 & 464.19 & 29.63 & 1.22 & 4.83 & 455.42 \\
		& load & 15.19 & 22.58 & 4.38 & 12.33 & 64.15 & 24.31 & 10.72 & 7.26 \\
		& total & 18.52 & 29.21 & 7.96 & 476.52 & 93.78 & 25.52 & 15.55 & 462.68 \\
```

Reply:
fixed that, looks good here, let me know when you try again.